packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -16,6 +15,8 @@
     display: flex;
     align-items: center;
     font-size: $iui-font-size;
+    min-height: $iui-component-height;
+    height: 100%;
```

Comment:
`flex-grow` I assume would not work, cause I need vertical size..
Have any better suggestions?

Reply:
ah ok, this is because we have a wrapper `<li>`. Maybe you could apply grid or flex on that. Or maybe it's fine this way because 100% height is controlled by the tallest element inside the `<ul>`